synth_tools/commands/agents.py

```diff
@@ -38,8 +39,16 @@ def get_agent(
     """
     api = ctx.find_object(APIs)
     for i in agent_ids:
+        try:
+            a = api.syn.agent(i)
+        except KentikAPIRequestError as e:
+            if e.response.status_code == 404:
+                typer.echo(f"Agent {i} does not exists")
+            else:
+                typer.echo(f"Got unexpected response from API:\n{e.response.text}")
```

Comment:
Print the whole text version of the exception: `fail(f"{e}")`. It contains also the request and status code, which may come handy when reporting the error to Kentik support.